fixtures/ios_app_with_signing/Project.swift

```diff
@@ -0,0 +1,13 @@
+import ProjectDescription
+
+let project = Project(name: "App",
+                      targets: [
+                          Target(name: "App",
+                                 platform: .iOS,
+                                 product: .app,
+                                 bundleId: "io.tuist.SignApp",
+                                 infoPlist: "Info.plist",
+                                 sources: "App/**",
+                                 dependencies: []))
```

Comment:
This project does not generate by default, there's an extra `)`

```bash
$ /usr/bin/xcrun swiftc --driver-mode=swift -suppress-warnings -I /Users/oliver/Library/Developer/Xcode/DerivedData/tuist-edomxougefoacwakknnztphpgmtm/Build/Products/Debug -L /Users/oliver/Library/Developer/Xcode/DerivedData/tuist-edomxougefoacwakknnztphpgmtm/Build/Products/Debug -F /Users/oliver/Library/Developer/Xcode/DerivedData/tuist-edomxougefoacwakknnztphpgmtm/Build/Products/Debug -lProjectDescription /Users/oliver/Development/Tuist/fixtures/ios_app_with_signing/Project.swift --tuist-dump
/Users/oliver/Development/Tuist/fixtures/ios_app_with_signing/Project.swift:11:51: error: expected ',' separator
                                 dependencies: []))
                                                  ^
                                                  ,
/Users/oliver/Development/Tuist/fixtures/ios_app_with_signing/Project.swift:11:51: error: expected expression in container literal
                                 dependencies: []))
                                                  ^
```

Reply:
Oops, forgot to check, good check 👍 This should have a fixture test sooner than later to catch these as soon as possible